docs/relnotes.md

```diff
@@ -15,7 +15,7 @@ This is the list of changes to Taipy releases as they were published.
 
    - The data node of a scenario or a pipeline can now be accessed at the scenario or pipeline levels.
    - When submitting a scenario, a pipeline or a task, a list of jobs or a Job will be returned.
-   - A submitted job can be canceled. When canceling the job, its subsequent jobs will be abandoned. When the cancel method is called on a running job, it cannot be canceled, but its subsequent jobs will be abandoned.
+   - A submitted job, a pending job or a blocked job can be canceled. When canceling the job, its subsequent jobs will be abandoned. On the other hand, running job, completed job, skipped job, failed job, canceled job and abandoned job cannot be canceled. When the cancel method is called on a running job, a completed job or a skipped job, its subsequent jobs will be abandoned while its status remains unchanged.
```

Comment:
```suggestion
   - A submitted job, a pending job or a blocked job can be cancelled. When cancelling the job, its subsequent jobs will be abandoned. On the other hand, running jobs, completing jobs, skipping jobs, failing jobs, cancelling jobs and abandoning jobs cannot be cancelled. When the cancelled method is called on a running job, a completed job or a skipped job, its subsequent jobs will be abandoned while its status remains unchanged.
```

Reply:
@florian-vuillemot Here I used comleted, skipped and failed to express a job that is finished. completing is like a job is being completed and not already completed. Same for skipping and failing. It's a bit weird in terms of grammar but I think it delivers better meanings

Reply:
Based on a suggestion from Fabien, I have fixed this part to use the `status` from Taipy instead of plain English to describe it. The benefit would be 1. grammartically correct, 2. conserve the correct meaning of the text :D